README.md

```diff
@@ -578,6 +579,17 @@ A curated list of awesome Java frameworks, libraries and software.
 * [Hibernate](http://hibernate.org/orm/) - Robust and widely used with an active community.
 * [MyBatis](http://www.mybatis.org/mybatis-3/) - Couples objects with stored procedures or SQL statements.
 
+## PaaS
+
+*Java platform as a service.*
+
+* [AWS Elastic Beanstalk ![c]](https://aws.amazon.com/elasticbeanstalk/) - AWS-based with support for Tomcat and Jetty.
+* [AWS Lambda ![c]](https://aws.amazon.com/lambda/) - Serverless computation.
+* [Google App Engine ![c]](https://cloud.google.com/) - PaaS on Google's infrastructure.
+* [Heroku ![c]](https://www.heroku.com/) - Abstract computing environments.
+* [Jelastic ![c]](https://jelastic.com/) - Support Tomcat, Jetty, GlassFish, JBoss, TomEE and WildFly.
```

Comment:
I think "Supports" sounds better than "Support" here (note the lack of _s_ at the end).